internal/node/controller/auth.go

```diff
@@ -1,4 +1,4 @@
-package api
+package controller
```

Comment:
doing this fixed a import cycle when I was trying to import `api` somewhere else, and also this probably belongs where it's used